testsuite/CHANGELOG

```diff
@@ -17,6 +17,7 @@ and https://github.com/MDAnalysis/mdanalysis/wiki/UnitTests
          tyler.je.reddy
 
   * 0.16
+    - Added unit tests for MDAnalysis.analysis.psa exception handling
```

Comment:
That's the testsuite `CHANGELOG`, which @orbeckst and others seemed to be ok with using for testing-related changes, but I'll go ahead and remove this entry and cease placing test entries in the testsuite `CHANGELOG` in the future.